### ISSUE TITLE
Pesquisa incorreta após pesquisa correta.

### ISSUE THREAD

**Issue by juliobguedes** [paulojbleitao/pokedex#9]
No fluxo de pesquisar e selecionar um pokemon corretamente e, em seguida, pesquisar por um pokemon inexistente, as informações do primeiro pokemon continuam na tela, mas a notificação de que o pokemon não foi encontrado toma o foco principal da exibição. O comportamento desejado é realmente esse?

Acredito que seria mais interessante manter as duas informações separadas, que achas?

**Comment by paulojbleitao** [comment]
Sim, o comportamento que eu pensei era esse mesmo :thinking: Não entendi muito bem o problema

**Comment by juliobguedes** [comment]
Acredito que o comportamento de mostrar que não foi encontrado um Pokemon com aquele nome poderia ser mostrado de outra forma, a exemplo de um toast. Ainda assim, permanecer na página do Pokemon que foi encontrado é uma boa, realmente.
